CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v31.0.0
+
+Kubernetes API Version: v1.31.1
```

Comment:
API changes from v1.31.1 are not included as code is generated for v1.31.0.

Reply:
Yes, the release tooling has a bug, fixed the CHANGELOG manually.